caffe2/CMakeLists.txt

```diff
@@ -211,10 +206,579 @@ if(NOT BUILD_ATEN_ONLY)
   endif()
 endif()
 
+
+# ==========================================================
+# formerly-libtorch
+# ==========================================================
+
+set(TORCH_SRC_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../torch")
```

Comment:
we should probably just lift this main folder up to avoid `..` but we can do it in a separate PR